### DISCUSSION TITLE
New sidebar display options

### DISCUSSION THREAD

**Discussion by MrBrax** [MrBrax/LiveStreamDVR#323] [News]
![1665144653398](https://user-images.githubusercontent.com/1517911/194561214-7750ae95-891c-4de4-9974-77c4372b1744.png)
![1665148096485](https://user-images.githubusercontent.com/1517911/194561219-7972eb7a-663d-4c30-945a-9c6c76a4ffa1.png)

Fields are now customizable to a certain degree (develop-ts). Want more fields to show up?

**Comment by Nincodedo** [comment]
Stream title would be a nice addition.
